agent/api/task/task.go

```diff
@@ -140,6 +140,22 @@ const (
 
 	// specifies awsvpc type mode for a task
 	AWSVPCNetworkMode = "awsvpc"
+
+	internalExecAgentNamePrefix          = "__internal__exec-agent"
```

Comment:
any reason why double underscores?

Reply:
I just wanted something unique that won't collide with other volume names. do you have suggestions?

Reply:
How about `execute-command-agent-internal` or `internal-execute-command-agent`?

As this isn't an environment variable, I see `awslogsCredsEndpointOpt = "awslogs-credentials-endpoint"` above and I think this is a nice convention.
Also we should be wary of calling this the "exec-agent" in code.  Note, we changed the references in all our designs to call it "execute-command-agent".

Reply:
I can do that, although I was trying to march some other "internal" stuff naming (e.g. [this](https://github.com/aws/amazon-ecs-agent/blob/478c4ec00a43b4713a0acc811cfd1e2d70307f63/agent/api/task/task.go#L64)), but I can go with whatever we decide it's best.

Reply:
Fixed: I went with `agent/config/config_windows.go`